source/common/http/http1/codec_impl.h

```diff
@@ -330,6 +330,7 @@ class ConnectionImpl : public virtual Connection, protected Logger::Loggable<Log
   Protocol protocol_{Protocol::Http11};
   const uint32_t max_headers_kb_;
   const uint32_t max_headers_count_;
+  const bool reject_unsupported_transfer_encodings_ : 1;
```

Comment:
nit: can you pack this into the protected bitfield above?

Reply:
✅